scripts/pull_stats.py

```diff
@@ -18,37 +19,41 @@
 urllib3.disable_warnings()
 
 if __name__ == "__main__":
+    for game in ['uya', 'dl']:
```

Comment:
I generally prefer iterating over a tuple for fixed collections to show immutability as my intent. In this case the list is never assigned to a variable so it cannot be mutated further. This is more of a suggestion to make the code more homogenous.